### ISSUE TITLE
question: disposing of CanvasDevice - will it release all the bitmaps that were created on it?

### ISSUE THREAD

**Issue by jtorjo** [microsoft/Win2D#804]
The documentation says "CanvasDevice.Dispose - Releases all resources used by the CanvasDevice."

I would then expect any CanvasBitmap (or any other resource) pertaining to this device to be properly disposed and its memory released. 

And, accessing any CanvasBitmap pertaining to this device after it's been disposed, it should throw an exception.

Are my assumptions correct?

**Comment by ChewWorker** [comment]
I believe that is correct.  Could write a small test to verify :)

**Comment by jtorjo** [comment]
Yes, that would be excellent! I'm not really sure how to do that in c# - like, what am i supposed to look for?
